src/View/Components/Card.php

```diff
@@ -80,4 +81,16 @@ public function personalization(): array
             ],
         ]);
     }
+
+    final public function transitions(): string
```

Comment:
Doesn't make sense to implement this as a method here, also as a final method. I'm thinking about providing the ability to personalize transitions via soft personalization in v2. I'll keep this in draft for a while, so I can come back here when I decide what to do exactly.